src/chat.rs

```diff
@@ -1839,7 +1840,7 @@ impl Chat {
     /// deltachat, and the data returned is still subject to change.
     pub async fn get_info(&self, context: &Context) -> Result<ChatInfo> {
         let draft = match self.id.get_draft(context).await? {
-            Some(message) => message.text,
+            Some(message) if self.can_send(context).await? => message.text,
```

Comment:
With this line you can __not__ set a draft while we are still waiting for securejoin and if chat is still in contact request phase. Also you can not set it if protection status is broken. Maybe this is too restrictive and we should only check `is_contact_in_chat` here aswell.